docs/user_guide/configuring.rst

```diff
@@ -19,20 +19,78 @@ doc path's _static folder, and use the following configuration:
 
    html_logo = "_static/logo.png"
 
-Configure social media buttons
-==============================
+Configure icon links
+====================
 
-If you'd like social media buttons to show up to the right of your nav bar, use the
+If you'd like icon links to show up to the right of your main navigation bar, use the
 following configuration:
 
 .. code:: python
 
    html_theme_options = {
-     "github_url": "https://github.com/<your-org>/<your-repo>",
-     "gitlab_url": "https://gitlab.com/<your-org>/<your-repo>",
-     "twitter_url": "https://twitter.com/<your-handle>",
+       ...
+       "icon_links": [
+           {
+               "name": "GitHub",
+               "url": "https://github.com/<your-org>/<your-repo>",
+               "icon": "fab fa-github-square",
+           },
+           {
+               "name": "GitLab",
+               "url": "https://gitlab.com/<your-org>/<your-repo>",
+               "icon": "fab fa-gitlab",
+           },
+           {
+               "name": "Twitter",
+               "url": "https://twitter.com/<your-handle>",
+               "icon": "fab fa-twitter-square",
+           },
+       ],
+       ...
    }
 
+
+The value of ``icon`` can be any full
+`FontAwesome 5 Free <https://fontawesome.com/icons?d=gallery&m=free>`__.
+In addition to the main icon class, e.g. ``fa-cat``, the "style" class must
+also be provided e.g. `fab` for *branding*, or `fas` for *solid*.
+
+
+.. Hint::
+
+   To get custom colors like "Twitter blue", use the following in your CSS,
+   e.g. ``custom.css``:
+
+   .. code:: css
+
+      i.fa-twitter-square:before {
+         color: #55acee;
+      }
```

Comment:
If we intend to keep that in our css, it might be worth mentioning that this is already done for twitter and github, so it's clear that you don't need to do the above manually for twitter/github?

Reply:
added in d616236d!